man/systemd-run.xml

```diff
@@ -244,7 +244,15 @@
 
         <para>When both <option>--pipe</option> and <option>--pty</option> are used in combination the more appropriate
         option is automatically determined and used. Specifically, when invoked with standard input, output and error
-        connected to a TTY <option>--pty</option> is used, and otherwise <option>--pipe</option>.</para></listitem>
+        connected to a TTY <option>--pty</option> is used, and otherwise <option>--pipe</option>.</para>
+
+        <para>When this option is used the original file descriptors <command>systemd-run</command> receives are passed
```

Comment:
Perhaps I’d’ve put a comma after “used”… but it’s not important, since the PR was already merged :)

Reply:
I think both options are OK. Modern usage seems to economize on commas and skip them in such cases. (I guess because that's how people speak, they don't pause before between "used" and "the".)